.github/dependabot.yml

```diff
@@ -13,6 +13,15 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    ignore:
+      # Only update them manually since updating them might break compatibility
+      - dependency-name: "numpy"
+      - dependency-name: "protobuf"
+    open-pull-requests-limit: 5
```

Comment:
Is this related to your change?

Reply:
Previously dependabot will automatically create unlimited PRs for PyPI package updates and this PR will set the threshold as 5 by `open-pull-requests-limit: 5`. I will mention it in PR title also.